server/schema.ts

```diff
@@ -0,0 +1,67 @@
+import { gql } from 'https://deno.land/x/graphql_tag@0.0.1/mod.ts';
+
+
+
+export const resolvers = {
+    Query: {
+       oneRocket: async (           
+           _parent: any,
+           args: any,
+           { denostore }: any,
+           info: any
+       ) => {
+        const results = await denostore.cache( 
```

Comment:
Could we please directly "return await denostore.cache" here instead of creating two variables called results to be returned? This way we can eliminate line 24.